src/network/clientpackethandler.cpp

```diff
@@ -1009,7 +1012,7 @@ void Client::handleCommand_AddParticleSpawner(NetworkPacket* pkt)
 	event.add_particlespawner.attached_id        = attached_id;
 	event.add_particlespawner.vertical           = vertical;
 	event.add_particlespawner.texture            = new std::string(texture);
-	event.add_particlespawner.id                 = id;
+	event.add_particlespawner.id                 = client_id;
```

Comment:
it's still the spawner ID, the reason the local variable was renamed because the function needs to handle both the local ID and the remote ID.